index.js

```diff
@@ -61,17 +71,41 @@ module.exports = {
             }
         }, options);
 
+        /** Plugins */
+        eleventyConfig.addPlugin(EleventyRenderPlugin);
+        eleventyConfig.addPlugin(pluginWebc);
+
         /** Filters */
         eleventyConfig.addFilter("formatDate", formatDateFilter);
         eleventyConfig.addFilter("isoDate", isoDateFilter);
         eleventyConfig.addFilter("limit", limitFilter);
-        eleventyConfig.addFilter("markdown", markdownFilter);
+        eleventyConfig.addFilter("markdown", function (value) {
+            // eslint-disable-next-line no-console
+            console.warn("This filter will be removed in a future version of eleventy-plugin-fluid. Use the renderString shortcode instead.");
+
+            const md = new MarkdownIt(options.markdown.options);
+            options.markdown.plugins.forEach(plugin => {
+                if (plugin) {
+                    md.use(require(plugin));
+                }
+            });
+
+            return md.render(value);
+        });
         eleventyConfig.addFilter("slug", () => {
             throw new Error("`slug` filter is no longer supported. Please use `slugify`.");
         });
         eleventyConfig.addFilter("split", splitFilter);
 
         /** Shortcodes */
+        eleventyConfig.addShortcode("renderString", async function (content, format) {
+            if (["html", "md", "webc", "11ty.js", "liquid", "njk", "hbs", "mustache", "ejs", "haml", "pug"].includes(format)) {
```

Comment:
Can we move this list of content/template types to the configuration. We can have these as the default, but that would provide a way for the integrator to add another one if they add more plugins for example.